tests/kernel/mbox/mbox_api/src/test_mbox_api.c

```diff
@@ -433,12 +548,12 @@ void test_mbox_kdefine(void)
  * - with same data.
  *
  */
+static ZTEST_BMEM char __aligned(4) buffer[8];
 void test_enhance_capability(void)
 {
 	info_type = ASYNC_PUT_GET_BUFFER;
 	struct k_msgq msgq;
 
-	ZTEST_BMEM char __aligned(4) buffer[8];
```

Comment:
I don't understand why we have to remove this line.

Reply:
I just move it to a static type because of compile error, not remove.